### ISSUE TITLE
Not receiving any authentication code

### ISSUE THREAD

**Issue by nixmeer** [LorenzHW/My-Telegrams#2]
Hello :) 

I tried to get this skill to work. When entering my Telegram (and mobile) number and asking Alexa to start the authentication, there is no code being sent to my phone or telegram. Why is that?

If I replace the country code with a 0, I get sometimes a message saying this number has no telegram account. 

I also tried to delete my account on your website which didn't seem to work. I can still login using the same google account

Hope this helps you to find the bug :)

**Comment by nixmeer** [comment]
Hello? Is there anybody with the same issue?

**Comment by LorenzHW** [comment]
Hi, sorry for the late response! I am currently not maintaining this project anymore, but thanks for the information. In case I will start working on it again I will keep it in mind.

**Comment by nixmeer** [comment]
Hi :) sorry to hear that :/ it’s a really useful project. Do you have any idea why this happens?

**Comment by LorenzHW** [comment]
Thank you! Yes it is. But also very time consuming.

So I just tested it as well and you are right. Alexa says: "You just received a code on your phone. ..." However, I am not getting a code as well. So the frontend is working fine. The [request](https://github.com/LorenzHW/My-Telegrams/blob/a37291a6334f64acd029d6338f54c0c870bd7622/lambda/py/skill/services/telethon_service.py#L23) for the code is sent to my backend. However, then something is happening in the backend. This code is not public though. So nothing much you can do. Sorry!

**Comment by LorenzHW** [comment]
Also the backend responds with the correct status code otherwise an exception would be raised and alexa would respond differently. Not entirely sure what is happening TBH. Maybe it just gets lost in the backend 🙈

**Comment by nixmeer** [comment]
Thank you so much for checking! 
Did you ever think about handing this project over to the official telegram? Maybe they‘re interested? Or maybe not since it’s Alexa and they’re concerned about security

**Comment by txkirrane** [comment]
Yeah, I've run into the same problem. Would it be possible for you to search up the confirmation code on your end? It would be greatly appreciated!
